manage.html

```diff
@@ -254,7 +251,7 @@ <h1 id="manage-heading" i18n-text="manageHeading"></h1>
     <button id="check-all-updates-force" class="hidden" i18n-text="checkAllUpdatesForce"></button>
   </p>
 
-  <p>
+  <p class="add-style-wrapper">
```

Comment:
It's a single element's class so I'm inclined to think we should use an id like we always did. What do you think? 

Everything else seems good.

Reply:
What about making it an id of "write-style-block"?

Reply:
I'd ilke to reuse the existing element's prefix `add-style`

Reply:
ping

Reply:
The existing element has id `add-style-label` so it'd be reasonable to use `add-style-wrapper` for the wrapper.